### DISCUSSION TITLE
My Self Introduction

### DISCUSSION THREAD

**Discussion by ProfBelts** [community/community#63499] [GitHub Education]
### Hi everyone,

Hi everyone,

I am Joseph June S. Beltran an incoming 4th Year College Student at Technological Institute of the Philippines. This might sound strange, but actually I am an AB English Major student. After some extensive soul searching I decided that I want to pursue a career in tech, thus leading me here in this platform.

**Comment by drdixit** [comment]
👋Hello @ProfBelts

**Comment by Miny320** [comment]
I am a full-stack developer from Japan.
This is my portfolio.
https://www.miny-dev.com/
I am currently working as a freelancer.
I would like to expand beyond the domestic market into the global market to gain experience and increase my income.
I am particularly interested in pioneering the Philippine market, so I would be very grateful if anyone could offer their assistance.
If you would like to collaborate with me on the Philippine market but also on generating revenue, we can discuss the details.
I have an idea.
